liota/lib/protocols/helix_protocol.py

```diff
@@ -128,125 +125,45 @@ def is_active(self):
         return self == self.proto.state
 
 
-class HandshakeAwaitingState(State):
-
+class HandshakeRequestedState(State):
```

Comment:
To get hello message response and transaction id used for further communication in case of WebSocket